proposals/dictionary-expressions.md

```diff
@@ -435,3 +464,5 @@ Working group recommendation: Use normal parsing here.  So this would be the sam
 What are the rules when types have multiple indexers and multiple impls of `IEnumerable<KVP<,>>`
 
 It would likely make sense to align with whatever comes out if you `foreach`ed the collection.
+
+TODO: Working group recomendation: Look at what happens with existing *collection expressions* and *collection types* with similar cases.  Follow that.  
```

Comment:
>Look at what happens with existing *collection expressions* and *collection types* with similar cases. [](http://example.com/codeflow?start=35&length=102)

A collection type with multiple implementations of `IEnumerable<T>` (and with no accessible `GetEnumerator()` method) cannot be used as a collection expression target type.